en/docs/learn/extension-points-for-oauth.md

```diff
@@ -229,28 +232,31 @@ If a supported grant type is to be modified you may implement a grant validator
 <tr class="even">
 <td>Other Available Implementations</td>
 <td><div class="content-wrapper">
-<ul>
-<li><p><code>                 org.wso2.carbon.identity.oauth.tokenprocessor.EncryptionDecryptionPersistenceProcessor                                 </code> Use this if you want to store access tokens, refresh tokens, authorization codes, and client secrets in an encrypted format using the OEAP (RSA/ECB/OAEPwithSHA1andMGF1Padding) algorithm. With this implementation you can use a valid token until it is either revoked or becomes invalid.</p>
-<p>!!! tip</p>
-    <p>OAuth2 token encryption protects OAuth2 access tokens, refresh tokens, consumer secrets, and authorization codes. To enable OAuth token encryption, configure the <code>                 TokenPersistenceProcessor          </code> as shown below in the <code>                 &lt;IS_HOME&gt;/repository/conf/deployment.toml        </code> file:</p>
+<pre><code>org.wso2.carbon.identity.oauth.tokenprocessor.EncryptionDecryptionPersistenceProcessor</code></pre>
+<p>Use this if you want to store access tokens, refresh tokens, authorization codes, and client secrets in an encrypted format using the OEAP (RSA/ECB/OAEPwithSHA1andMGF1Padding) algorithm. With this implementation you can use a valid token until it is either revoked or becomes invalid.
+<div class="admonition tip">
+<p class="admonition-title">Tip</p>
+    <p>OAuth2 token encryption protects OAuth2 access tokens, refresh tokens, consumer secrets, and authorization codes. To enable OAuth token encryption, configure the <code>                 TokenPersistenceProcessor          </code> as shown below in the <code>                 &lt;IS_HOME&gt;/repository/conf/deployment.toml        </code> file:
```

Comment:
Shall we mention TokenPersistenceProcessor as token_persistance_processor?

Reply:
Fixed in 6cad0bf12e3eca4ca70d9dea7b04128ea0b02680